src/js/views/programs/shared/components/published_component.js

```diff
@@ -19,41 +19,61 @@ const PublishedTemplate = hbs`<span class="{{ className }}">{{far icon}}<span>{{
 
 const PublishedStates = [
   {
-    id: PUBLISH_STATE_STATUS.DRAFT,
     icon: 'pen-to-square',
     className: 'program-action--draft',
     name: i18n.draftText,
+    published: false,
+    behavior: PROGRAM_BEHAVIORS.STANDARD,
   },
   {
-    id: PUBLISH_STATE_STATUS.PUBLISHED,
     icon: 'circle-play',
     className: 'program-action--published',
     name: i18n.publishedText,
+    published: true,
+    behavior: PROGRAM_BEHAVIORS.STANDARD,
   },
   {
-    id: PUBLISH_STATE_STATUS.CONDITIONAL,
     icon: 'circle-pause',
     className: 'program-action--conditional',
     name: i18n.conditionalText,
+    published: true,
+    behavior: PROGRAM_BEHAVIORS.CONDITIONAL,
+  },
+  {
+    icon: 'bolt',
+    className: 'program-action--automated',
+    name: i18n.automatedText,
+    published: true,
+    behavior: PROGRAM_BEHAVIORS.AUTOMATED,
   },
 ];
 
 export default Droplist.extend({
   isCompact: false,
   initialize(options) {
-    this.mergeOptions(options, ['status', 'isPublishedDisabled', 'isConditionalAvailable']);
+    const { published, behavior } = options;
+    this.mergeOptions(options, ['isPublishDisabled', 'isConditionalAvailable']);
```

Comment:
this used the option `isPublishedDisabled` which wasn't in use because it's `isPublishDisabled`, but it is consistent with the `isCond...` and doesn't cause coverage issues so I kept it